examples/list.py

```diff
@@ -0,0 +1,14 @@
+from ollama import list
+from ollama._types import ListResponse
+
+response: ListResponse = list()
+
+for model in response.models:
+  if model.details:
+    print(f'Name: {model.model}')
```

Comment:
fstrings are unnecessary here

```suggestion
    print('Name:', model.model)
```

Reply:
For readability, every line other than `Name` should be indented